kolibri/logger/apps.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+from django.apps import AppConfig
+
+
+class KolibriLoggerConfig(AppConfig):
+    name = 'kolibri.logger'
+    label = 'logger'
+    verbose_name = 'Kolibri Logger'
+
+    def ready(self):
+        from .signals import login_session_receiver, content_session_receiver
```

Comment:
Register decorated signal callbacks just by importing them, and then assert them to satisfy our linter.